app/__init__.py

```diff
@@ -12,7 +12,11 @@
     nav.Item('Josue', 'index', {'person': 'Josue'}),
     nav.Item('Carolina', 'index', {'person': 'Carolina'})])
 
-@app.route('/', defaults={'person':'Miyabi'})
+
+@app.errorhandler(404)
+def not_found(e):
```

Comment:
I prefer to use the full word error instead of  `e` since sometimes when we deal with frontend we have events as well and that could be an `e` too. What do you think?